model/vm_pool.rb

```diff
@@ -13,13 +13,14 @@ class VmPool < Sequel::Model
 
   def pick_vm
     DB.transaction do
```

Comment:
@jeremyevans Actually, respirate runs all labels within a transaction, making the transaction here redundant and unnecessary. The `GithubRunner.allocate_vm` label and the `GitHubRunner.pick_vm` method both operate within a transaction.

https://github.com/ubicloud/ubicloud/blob/8731845a4b463eac77c28400eefc7d0aa4ef8f18/model/strand.rb#L43

Reply:
Understood, so this is safe in actual usage, it just looks unsafe because of this transaction call.

Reply:
Removed the DB.transaction, I agree that it is confusing